packages/theming/win32-theme/src/NativeModule/getThemingModule.native.ts

```diff
@@ -6,10 +6,10 @@ declare module 'react-native' {
   interface ITurboModuleRegistry {
     get: (name: 'Theming') => OfficeThemingModule & EventSubscriptionVendor;
   }
-  const TurboModuleRegistry: ITurboModuleRegistry;
+  const win32TurboModuleRegistry: ITurboModuleRegistry;
 }
 
-import { TurboModuleRegistry } from 'react-native';
+import { win32TurboModuleRegistry } from 'react-native';
```

Comment:
I don't think it needs to be. Before in RN 0.64 we were declaring TurboModuleRegistry. But when moving onto RN 0.66 I would get an error saying it has already been declared in the RN 0.66 types. So I had it named win32TurboModuleRegistry as an initial attempt to try to resolve that issue but that actually caused the tester app to break.

So now I'm just importing the  TurboModuleRegistry  from react native instead of declaring it as we did previously. Not sure of the implications of this though.